mappings/net/minecraft/inventory/Inventory.mapping

```diff
@@ -1,28 +1,45 @@
 CLASS net/minecraft/class_1263 net/minecraft/inventory/Inventory
-	METHOD method_18861 countInInv (Lnet/minecraft/class_1792;)I
+	METHOD method_18861 count (Lnet/minecraft/class_1792;)I
+		COMMENT Returns the number of times the specified item occurs in this inventory across all stored stacks.
 		ARG 1 item
-	METHOD method_18862 containsAnyInInv (Ljava/util/Set;)Z
+	METHOD method_18862 containsAny (Ljava/util/Set;)Z
+		COMMENT Determines whether this inventory contains any of the given candidate items.
 		ARG 1 items
 	METHOD method_5431 markDirty ()V
-	METHOD method_5432 onInvClose (Lnet/minecraft/class_1657;)V
+	METHOD method_5432 onClose (Lnet/minecraft/class_1657;)V
 		ARG 1 player
-	METHOD method_5434 takeInvStack (II)Lnet/minecraft/class_1799;
+	METHOD method_5434 removeStack (II)Lnet/minecraft/class_1799;
+		COMMENT Removes a specific number of items from the given slot.
+		COMMENT
+		COMMENT @return the removed items as a stack
 		ARG 1 slot
 		ARG 2 amount
-	METHOD method_5435 onInvOpen (Lnet/minecraft/class_1657;)V
+	METHOD method_5435 onOpen (Lnet/minecraft/class_1657;)V
 		ARG 1 player
-	METHOD method_5437 isValidInvStack (ILnet/minecraft/class_1799;)Z
+	METHOD method_5437 isValid (ILnet/minecraft/class_1799;)Z
+		COMMENT Returns whether the given stack is a valid for the indicated slot position.
 		ARG 1 slot
 		ARG 2 stack
-	METHOD method_5438 getInvStack (I)Lnet/minecraft/class_1799;
+	METHOD method_5438 getStack (I)Lnet/minecraft/class_1799;
+		COMMENT Fetches the stack currently stored at the given slot. If the slot is empty,
+		COMMENT or is outside the bounds of this inventory, returns see {@link ItemStack.EMPTY}.
 		ARG 1 slot
-	METHOD method_5439 getInvSize ()I
-	METHOD method_5441 removeInvStack (I)Lnet/minecraft/class_1799;
+	METHOD method_5439 size ()I
+	METHOD method_5441 removeStack (I)Lnet/minecraft/class_1799;
+		COMMENT Removes the stack currently stored at the indicated slot.
+		COMMENT
+		COMMENT @return the stack previously stored at the indicated slot.
 		ARG 1 slot
-	METHOD method_5442 isInvEmpty ()Z
-	METHOD method_5443 canPlayerUseInv (Lnet/minecraft/class_1657;)Z
+	METHOD method_5442 isEmpty ()Z
+	METHOD method_5443 canStayOpen (Lnet/minecraft/class_1657;)Z
+		COMMENT Controls whether this inventory can remain open when accessed by a player.
```

Comment:
I think that `canPlayerUse` is still better. `canStayOpen` (and the javadoc) sounds like there's another method for checking if it can initially be opened, and this one is checked after that.